services/core-web/src/components/modalContent/AddBondModal.js

```diff
@@ -14,20 +15,24 @@ const propTypes = {
   title: PropTypes.string.isRequired,
   bondTypeOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
   provinceOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
+  bondDocumentTypeDropDownOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
   permitGuid: PropTypes.string.isRequired,
-  bond: CustomPropTypes.bond.isRequired,
+  mineGuid: PropTypes.string.isRequired,
+  bond: CustomPropTypes.bond,
   editBond: PropTypes.bool,
 };
 
 const defaultProps = {
+  bond: {},
   editBond: false,
 };
 
 export const AddBondModal = (props) => {
-  const handleAddBond = (values) =>
-    props.editBond
+  const handleAddBond = (values) => {
```

Comment:
the change shouldn't be needed as it was an implicit return, but it doesn't matter.

Reply:
I put it back to normal, can't remember why I changed it.